src/components/Poll.vue

```diff
@@ -74,6 +74,10 @@ export default {
       document.getElementById('add-option-input').focus()
     },
 
+    editOption(newOption) {
+      this.options = this.options.map((option) => { return option.id === newOption.id ? newOption : option })
```

Comment:
if the arrow function is only one line you can skip the `return` statement and leave it like this:

`this.options.map(option => option.id === newOption.id ? newOption : option)`

Reply:
Nice, I like it.